libosmscout-client/src/osmscoutclient/MapManager.cpp

```diff
@@ -43,6 +43,11 @@ CancelableFuture<bool> MapManager::LookupDatabases()
     std::vector<std::filesystem::path> databaseFsDirectories;
 
     for (const auto &lookupDir:databaseLookupDirs){
+      if (!std::filesystem::exists(lookupDir) || !std::filesystem::is_directory(lookupDir)) {
+        osmscout::log.Warn() << "Lookup dir" << lookupDir << "doesn't exist";
```

Comment:
You need to convert lookupDir to string by `lookupDir.string()` explicitly, because there is just implicit conversion to "native path representation". And it is wstring on MSVC.

Reply:
Done